lib/streamlit/DeltaGenerator.py

```diff
@@ -487,6 +487,10 @@ def markdown(self, element, body, unsafe_allow_html=False):
         body : str
             The string to display as Github-flavored Markdown. Syntax
             information can be found at: https://github.github.com/gfm.
+
+            Emoji syntax is supported,
+            more information: https://www.webfx.com/tools/emoji-cheat-sheet/
```

Comment:
Better doc:

```
This also supports:
* Emoji shortcodes, such as `:+1:`  and `:sunglasses:`. For a list of all supported codes, see https://www.webfx.com/tools/emoji-cheat-sheet/.
* LaTeX expressions, by just wrapping them in "$" or "$$" (the "$$" must be on their own lines). Supported LaTeX functions are listed at https://katex.org/docs/supported.html.
```

And then remove the two paragraphs below too (the "Inline and block (...)" one and the also "The body also (...)")